pkgs/applications/video/kodi/addons/sponsorblock/default.nix

```diff
@@ -0,0 +1,29 @@
+{ lib, buildKodiAddon, fetchFromGitHub, six, requests }:
+buildKodiAddon rec {
+  pname = "sponsorblock";
+  namespace = "script.service.sponsorblock";
+  version = "0.5.0";
+
+  src = fetchFromGitHub {
+    owner = "siku2";
+    repo = namespace;
+    rev = "v${version}";
+    hash = "sha256-IBgh2kdPgCy+HHrR7UZxTgjF1LR77ABGlUp3PgaobNM=";
+  };
+
+  propagatedBuildInputs = [
+    six
+    requests
+  ];
+
+  passthru = {
+    pythonPath = "resources/lib";
+  };
+
+  meta = with lib; {
+    homepage = "https://github.com/siku2/script.service.sponsorblock";
+    description = "An unofficial port of the SponsorBlock browser extension. It works as an extension to the YouTube Plugin. Once installed, the add-on will automatically skip sponsor segments in all YouTube and Invidious videos you watch.";
```

Comment:
`description` should be very short, please see other packages for examples
you want `longDescription`

Reply:
Thanks you for your advice. I have shortened the description.